components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -152,6 +156,20 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 		this.addEventListener('keypress', stopPropagation);
 
 		this.__isChildView();
+
```

Comment:
Since this code is running in `firstUpdated` it's not going to work nicely if the view is disconnected and re-connected. But neither are all the event listeners being wired up above, so I just added to the problem for now. :)

Reply:
I think the ones immediately above aren't removed in disconnect, and the ones that are removed in the disconnected callback are added in the connected callback.  Two place this might be a concern are cards and list-items where those elements can be drag & dropped.

Reply:
Ok good call. I've moved the code around such that the `IntersectionObserver` now works when it's disconnected and then re-connected. I tested it in drag & drop lists that have menus, as well as the my-courses widget when the tiles move around.

Also tested out the mobile menu -- works well.